emote/algorithms/generic_agent_proxy.py

```diff
@@ -0,0 +1,105 @@
+from __future__ import annotations
+
+import numpy as np
+import torch
+
+from torch import nn
+
+from emote.proxies import AgentProxy
+from emote.typing import AgentId, DictObservation, DictResponse, EpisodeState
+from emote.utils.spaces import MDPSpace
+
+
+class GenericAgentProxy(AgentProxy):
```

Comment:
I do not think that GenericAgentProxy belongs to algorithms. I'd suggest to keep "emote/algorithms" for implementing methods such as SAC and DQN. And, putting generic codes into corresponding emote folders. As an example, we can have a python file for proxies and put this into the proxies folder.

Reply:
Good point, changed that!